python-sdk/nuscenes/eval/detection/algo.py

```diff
@@ -98,7 +98,11 @@ def accumulate(gt_boxes: EvalBoxes,
             match_data['trans_err'].append(center_distance(gt_box_match, pred_box))
             match_data['vel_err'].append(velocity_l2(gt_box_match, pred_box))
             match_data['scale_err'].append(1 - scale_iou(gt_box_match, pred_box))
-            match_data['orient_err'].append(yaw_diff(gt_box_match, pred_box))
+
+            # Barrier orientation is only determined up to 180 degree. (For cones orientation is discarded later)
```

Comment:
@sergi-nutonomy  is now checking that barriers are annotated systematically (with orientation accurate up to 180 degrees).

Reply:
Out of 30 random samples. 
13 samples'  orientations are facing sideways 
![image](https://user-images.githubusercontent.com/46339400/54408223-0892b700-471c-11e9-9a45-8c9b6fc30ea2.png)

17 samples' orientations are facing the road body
![image](https://user-images.githubusercontent.com/46339400/54408294-45f74480-471c-11e9-987a-ec03a4a2ef88.png)

Reply:
Thanks, @sergi-nutonomy. I suggest that either
- We measure orientation up to 90 degrees or
- We ignore orientation of barriers.
@oscar-nutonomy

Reply:
Great catch guys. Per off-line discussion we will patch the DB to fix the root cause of this.